### ISSUE TITLE
Need a way to specify conda channels

### ISSUE THREAD

**Issue by pitrou** [Anaconda-Server/anaconda-build#77]
If some of the engines in your matrix need version-specific dependencies, and those dependencies are not in the main conda channels, you are stuck because you cannot set channels at this point.

An example is the Numba build matrix, where "singledispatch" is available in the numba channel, but not in the main channel:

```
engine:
 - python=2.6 argparse funcsigs unittest2 singledispatch
 - python=2.7 funcsigs singledispatch
 - python=3.3 singledispatch
 - python=3.4
```

There should be a binstar.yml option to enable additional channels. That would also be more user-friendly than using a "conda config" command line.

**Comment by richardotis** [comment]
Does putting `conda build -c [channel] path/to/recipe` in your .binstar.yml do what you want?

**Comment by stephenakearns** [comment]
Add to Epic: productizing the build system

**Comment by stephenakearns** [comment]
Hi @pitrou - can you take a look at #14 and see if this is a duplicate? If so, can you close this and update the description of #14 if needed?

**Comment by dsludwig** [comment]
This has been added recently: http://docs.anaconda.org/build.html#Install_Channels (the channels will be added before the environment is created). I don't think it has been released yet (as of 0.13.2)
